### DISCUSSION TITLE
Add Air Purifier device

### DISCUSSION THREAD

**Discussion by normanr** [andrei-tatar/node-red-contrib-smartnora#91] [Ideas]
I have an air purifier that I can control with an IR blaster and would like to add it to Google Home. Currently I think I'd have to add it as a Fan, but it would be nice to add it as an Air Purifier. Thoughts?

**Comment by andrei-tatar** [comment]
@normanr it does seem the fan is the most similar device. I guess the node from node-red could be extended to support other device types as well (if there are any that use FanSpeed/OnOff traits).

**Comment by normanr** [comment]
There's an [Air Purifier](https://developers.google.com/assistant/smarthome/guides/airpurifier) device in the Smart Home schema, so why not just add a new node for that device?

**Comment by andrei-tatar** [comment]
@normanr version `1.5.0` comes with a new node that supports Air Purifier device.

**Reply by normanr** [reply]
Thanks! Unsurprisingly the only change is the icon on the UI. It would be nice if Google improved their UI for Fans/Air Purifiers/Air Conditioners, but at least the voice control works.

**Comment by normanr** [comment]
Note: docs need updating to mention this new device
